feature/permissions/src/main/java/com/google/jetpackcamera/permissions/PermissionsViewModel.kt

```diff
@@ -25,6 +25,7 @@ import dagger.assisted.Assisted
 import dagger.assisted.AssistedFactory
 import dagger.assisted.AssistedInject
 import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlin.collections.removeFirst as ktRemoveFirst
```

Comment:
nit: add a comment linking to https://issuetracker.google.com/348683480 to explain why we have to do this.